alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/rest/RestClient.java

```diff
@@ -114,13 +92,13 @@ public void setProviderConfiguration(ProviderConfig providerConfiguration) throw
             SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(
                     sslContext,
                     NoopHostnameVerifier.INSTANCE);
-            CloseableHttpClient httpClient = HttpClients
+            httpClient = HttpClients
                     .custom()
```

Comment:
To be able to deal with proxies defined in system properties, would need to replace **HttpClients.custom()** by
**HttpClientBuilder.create().useSystemProperties()**